### PR TITLE
241-control-metrics

### PR DESCRIPTION
**Pull request by idruzhitskiy**

### CODE REVIEW

**Comment by idruzhitskiy** [comment]
closes #253

File: config-examples/node.yaml [hunk 1]
```diff
@@ -64,9 +64,13 @@ pearl:
     # each thread will wait this period if another thread creating pearl. required for 'pearl'
     create_pearl_wait_delay: 100ms
 
-# optional, send metrics
+# send metrics
```
[the remark this answers is not in the document]
Comment (idruzhitskiy) [reply config-examples/node.yaml:67]: Removed *test.yaml files

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,7 +4,11 @@ Bob versions changelog
 
 ## [Unreleased]
 #### Added
+<<<<<<< HEAD
+- Add the ability to disable metrics ([#241](https://github.com/qoollo/bob/pull/253))
+=======
 - Use default config in dockerfiles ([#290](https://github.com/qoollo/bob/pull/309))
+>>>>>>> 73a010a84c9b05ffa2be4d7175d5d7ea32475e9c
```
Comment (ikopylov) [inline CHANGELOG.md:11]: This merge conflict should be fixed

File: bob-common/src/configs/node.rs [hunk 3]
```diff
@@ -141,12 +143,20 @@ impl MetricsConfig {
...
+    }
+
+    pub(crate) fn graphite(&self) -> Option<&str> {
+        self.graphite.as_deref()
     }
 
     fn check_unset(&self) -> Result<(), String> {
-        if self.graphite == PLACEHOLDER {
+        if self.graphite_enabled && self.graphite.is_none() {
             let msg = "some of the fields present, but empty".to_string();
```
Comment (ikopylov) [inline bob-common/src/configs/node.rs:160]: Can you change the message to a more understandable one?

**Pull request merged**